dj_rql/filter_cls.py

```diff
@@ -6,6 +6,8 @@
 from datetime import datetime
 from uuid import uuid4
 
+from cachetools import LFUCache
```

Comment:
https://cachetools.readthedocs.io/en/stable/#cache-implementations
```Please be aware that all these classes are not thread-safe. Access to a shared cache from multiple threads must be properly synchronized, e.g. by using one of the memoizing decorators with a suitable lock object```

RQLFilterBackend._CACHES is global. Could it be a problem?

Reply:
Discussed. Will keep without locks for now.